LCLSGeneral/LCLSGeneral/DUTs/DUT_EPS.TcDUT

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.12">
+  <DUT Name="DUT_EPS" Id="{0134451e-57c2-478a-9365-5d6570cca7b2}">
+    <Declaration><![CDATA[TYPE DUT_EPS :
+STRUCT
+    // Contains EPS flags
+    {attribute 'pytmc' := '
+        pv: EPS:nFlags
+        io: i
+        field: DESC Contains EPS flags
+    '}
+    nFlags: UDINT := 16#FFFFFFFF;
+
+    // Desciption of values nFlags contains
+    {attribute 'pytmc' := '
+        pv: EPS:sFlagDesc
+        io: i
+        field: DESC of nFlag variable
+    '}
+    sFlagDesc: STRING;
```

Comment:
This is missing some documentation about how it's expecting descriptions split by semicolons as used in https://github.com/pcdshub/pcdswidgets/pull/82/

This answers my question about "how would a screen know how many bits are used"

Reply:
Food for thought: would it be better to have an explicit integer PV that tells us how many bits are active? Maybe even a mask that tells us which bits were active? Or is that information redundant?